js/signup.js

```diff
@@ -0,0 +1,73 @@
+let signupEmail = document.querySelector('#signup-email');
+let signupNickname = document.querySelector('#signup-nickname');
+let signupPassword = document.querySelector('#signup-password');
+let signupCheckPassword = document.querySelector('#signup-check-password');
+let signupBtn = document.querySelector('.signup-btn');
+
+
+function inputSignupEmail(){
+  activeSignupBtn()
+}
+
+function inputSignupNickname(){
+  activeSignupBtn()
+}
+
+function inputSignupPassword(){
+  activeSignupBtn()
+}
+
+function inputSignupCheckPassword(){
+  activeSignupBtn()
+}
+
+function activeSignupBtn(){
+  if( signupEmail.value.length > 0 &&
+      signupNickname.value.length > 0 &&
+      signupPassword.value.length > 0 &&
+      signupCheckPassword.value.length > 0 &&
+      signupPassword.value === signupCheckPassword.value
+  ) {
+    signupBtn.classList.add('active');
+  }
+}
```

Comment:
아래처럼 toggle 을 사용하는 방법도 있습니다.

https://developer.mozilla.org/ko/docs/Web/API/Element/classList#toggle_string_force_


```suggestion

function activeSignupBtn() {
  const isValid = signupEmail.value.length > 0 &&
                  signupNickname.value.length > 0 &&
                  signupPassword.value.length > 0 &&
                  signupCheckPassword.value.length > 0 &&
                  signupPassword.value === signupCheckPassword.value;

  signupBtn.classList.toggle('active', isValid);
}

```